apps/passport-server/src/services/telegramService.ts

```diff
@@ -60,19 +61,24 @@ const adminBotChannel = "Admin Central";
 
 export class TelegramService {
   private context: ApplicationContext;
-  private bot: Bot<BotContext>;
+  private authBot: Bot<BotContext>;
+  private anonBot: Bot<BotContext>;
   private rollbarService: RollbarService | null;
+  private anonBotExists: boolean;
```

Comment:
I don't think we should use a class property for this. For example, what if authBot's ID changes to match anonBot? Then this variable would be out of date.

We can just use a local variable or have a anonBotExists() function